src/contexts/EthereumContext.tsx

```diff
@@ -87,7 +94,9 @@ interface EthereumContextType {
   connect: () => Promise<void>
   disconnect: () => void
   useOnboard: boolean
-  toggleOnboard: () => void // Function to toggle between Web3-Onboard and regular provider
+  toggleOnboard: () => void // Toggle between Web3-Onboard and regular provider
+  signInWithEthereum: () => Promise<void>
+  verified: boolean // Check if user is signed in
```

Comment:
`isVerified`

https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig